### ISSUE TITLE
testing: fuzzing: no string comparison instrumentation?

### ISSUE THREAD

**Issue by Crypt-iQ** [golang/go#50231]
`go1.18beta1` fuzzer is unable to figure out "tomatos" is a crasher after almost 40M iterations. Is string comparison not instrumented in go1.18 beta or perhaps there is a flag that I am missing? I was able to write a very similar test case with dvyukov's go-fuzz and it crashes in about 25 fuzzing iterations.
```
func FuzzThing(f *testing.F) {
	f.Fuzz(func(t *testing.T, b []byte) {
		if len(b) < 6 {
			return
		}

		if string(b) == "tomatos" {
			t.Fatalf("error!")
		}
	})
}
```

**Comment by cherrymui** [comment]
We don't really use our issue tracker for questions. More people will see your question, and you will get better answers, if you ask on a forum rather than on the issue tracker. See https://golang.org/wiki/Questions. Thanks.

cc @golang/fuzzing in case this is something the implementation missed.

**Comment by rolandshoemaker** [comment]
String comparison instrumentation is not present. It is one of the additional fuzzing strategies discussed in #46507, and is likely to be included in 1.19 as we continue to develop the fuzzing engine.
